charts/k8ssandra-cluster/templates/cassdc.yaml

```diff
@@ -33,10 +34,31 @@ spec:
       resources:
         requests:
           storage: {{ .Values.k8ssandra.cassandraLibDirVolume.size | default "5Gi" }}
-  config:    
+  config:
+    cassandra-yaml:
+{{- if .Values.k8ssandra.configuration.auth }}
+      authenticator: PasswordAuthenticator
+      authorizer: CassandraAuthorizer
+{{- else }}
+      authenticator: AllowAuthenticator
+      authorizer: AllowAuthorizer
+{{- end }}
+      role_manager: CassandraRoleManager
+      roles_validity_in_ms: {{ .Values.k8ssandra.configuration.auth.caches.rolesValidityMillis }}
```

Comment:
Is this intended that even if auth is set to false we still push these values?

Reply:
Yes, it is intended so as to avoid more conditional logic. The corresponding properties are always set in cassandra.yaml.